pkg/operator/starter.go

```diff
@@ -80,6 +80,8 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 			"csidriver.yaml",
 			"controller_sa.yaml",
 			"node_sa.yaml",
+			"service.yaml",
+			"servicemonitor.yaml",
```

Comment:
Didn't you want to use the new `WithServiceMonitorController()` instead?

https://github.com/openshift/library-go/pull/1042/files#diff-ca366a7dedcbe10d49342262a66fa7737b8205813d6bb29224aed6f828713cdeR192-R209

Reply:
I think I use it. Is it some old version of the PR?

Reply:
Seems like a caching issue